packages/data-stores/src/reader/queries/use-post-subscriptions-query.ts

```diff
@@ -57,13 +59,25 @@ const usePostSubscriptionsQuery = ( {
 	const outputData = useMemo( () => {
 		// Flatten all the pages into a single array containing all subscriptions
 		const flattenedData = data?.pages?.map( ( page ) => page.comment_subscriptions ).flat();
+
 		// Transform the dates into Date objects
 		const transformedData = flattenedData?.map( ( comment_subscription ) => ( {
 			...comment_subscription,
 			subscription_date: new Date( comment_subscription.subscription_date ),
 		} ) );
-		return transformedData?.filter( filter ).sort( sort );
-	}, [ data, filter, sort ] );
+
+		const searchTermLowerCase = searchTerm.toLowerCase();
+
+		const searchFilter = ( item: PostSubscription ) =>
+			searchTermLowerCase ? item.post_title.includes( searchTermLowerCase ) : true;
```

Comment:
The search doesn't return all the results because we are not turning the `post_title` into lowercase. This small change should do the trick:

```suggestion
			searchTermLowerCase ? item.post_title.toLowerCase().includes( searchTermLowerCase ) : true;
```